tools/SRS_tools/.shed.yml

```diff
@@ -0,0 +1,14 @@
+categories:
+    - Ecology
+owner: ecology
+remote_repository_url: https://github.com/Marie59/Sentinel_2A/SRS_tools
+long_description: |
+  Compute biodiveristy indicators from Sentinel 2 remote sensing data
+type: unrestricted
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
+  description_template: "Wrapper for spatial remote sensing tool: {{ tool_name }}."
+suite:
+  name: "Sentinel_2A"
+  description: "A suite of indicators for remote sensing data"
+  type: repository_Sentinel_2A
```

Comment:
this is probably a wrong type

Reply:
I put unrestricted, but I am really not sure. I don't quite understand what I should write.